draft-ietf-tls-tls13.md

```diff
@@ -3335,7 +3335,7 @@ and for a client signature is "TLS 1.3, client CertificateVerify". A
 hash of the handshake messages is signed rather than the messages themselves
 because the digitally-signed format requires padding and context bytes at the
 beginning of the input. Thus, by signing a digest of the messages, an
-implementation need only maintain one running hash per hash type for
+implementation only needs to maintain a single running hash per hash type for
```

Comment:
Both versions are correct. The suggested replacement could be considered simpler phrasing, which looks fine to me.